package.json

```diff
@@ -1,28 +1,30 @@
 {
   "name": "eslint-config-smartprocure",
-  "version": "2.0.1",
+  "version": "3.0.0",
   "description": "SmartProcure's ESLint configuration",
   "scripts": {
     "lint": "eslint .",
-    "lint:ci": "npm run lint"
+    "lint:fix": "npm run lint -- --fix",
+    "lint:ci": "npm run lint -- -o lint-results.json -f json"
   },
   "main": "index.js",
   "license": "MIT",
   "repository": {
     "type": "git",
     "url": "https://github.com/smartprocure/eslint-config-smartprocure.git"
   },
-  "devDependencies": {
-    "eslint": "^4.5.0"
-  },
   "dependencies": {
-    "eslint-plugin-import": "^2.7.0",
-    "eslint-plugin-jest": "^21.0.0",
-    "eslint-plugin-lodash": "^2.4.5",
-    "eslint-plugin-lodash-fp": "^2.1.3",
-    "eslint-plugin-mocha": "^4.11.0",
-    "eslint-plugin-react": "^7.3.0",
-    "eslint-plugin-smartprocure": "^1.0.2",
-    "lodash": "^4.17.4"
+    "eslint-plugin-import": "2.25.3",
```

Comment:
Since this is a dependency of other projects, will locking these down so strictly potentially cause conflicts with other dependencies with this package is added?

Reply:
good question. I don't think so?
If a project is using this package, they should not be adding these deps separately?